walkthroughs/week-2-web-development/examples/stanley/stanley-portfolio.iml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do you know why this file and others are added as new in this PR? This file is part of the week 2 example code.

Reply:
I believe It might've been caused by me got stashing and popping the stash ?

Reply:
I'm still not sure where they came from though. Can we remove them from the PR?